supabase/migrations/20251204000003_create_account_workspace_ids.sql

```diff
@@ -0,0 +1,68 @@
+-- Create account_workspace_ids table
+-- Links owner accounts to workspace accounts
+-- Similar structure to account_artist_ids (without pinned column)
+-- Used to identify which accounts are workspaces (vs artists)
+
+CREATE TABLE "public"."account_workspace_ids" (
```

Comment:
small note, I'd prefer this to be called

```suggestion
CREATE TABLE "public"."account_workspaces" (
```